src/interface.js

```diff
@@ -0,0 +1,76 @@
+$(document).ready(function(){
+  var bowling = new Game();
+
+  var addScore = function(){
+
+    if (bowling.scoreArray.length <= 21) {
+
+    for (var i = 1; i <= 10; i ++){
+      $('#f' + i + 'score').text(bowling.scoreTotal(i));
+    }
+
+    for (var j = 0; j < 21; j ++){
+      $('.box' + (j+ 1)).text(bowling.screenArray[j]);
+    }
+  }
+  };
+
+  $('#newgame').click(function() {
+    location.reload();
+  });
+
+  $('#btn0').click(function(){
```

Comment:
This doesn't feel DRY - what about adding a class 'score-button' to all of the score buttons and doing something like:

```
for (var i = 0; i < 10; i++) {
  $('.score-button')[i].click(function() {
    bowling.pinsDroppedEachRoll(i);
    addScore()
  });
}
```